views/login/questions/bankruptcy.pug

```diff
@@ -0,0 +1,55 @@
+extends ../../base
+
+block variables
+  -var title = __('Enter details about your bankruptcy assignment')
+  -var trusteeFirstName = hasData(body, 'trusteeFirstName') ? body.trusteeFirstName : ''
+  -var trusteeLastName = hasData(body, 'trusteeLastName') ? body.trusteeLastName : ''
+  -var dobDay = hasData(body, 'dobDay') ? body.dobDay : ''
+  -var dobMonth = hasData(body, 'dobMonth') ? body.dobMonth : ''
+  -var dobYear = hasData(body, 'dobYear') ? body.dobYear : ''
+
+block content
+
+  h1 #{title}
+
+  div
+    form.cra-form(method='post')
+
+      div.dob-container(class={['has-error']: errors && errors.dobDay})
+        fieldset.fieldset(role="group", aria-describedby="dobHint")
+          legend
+            h2.bankruptcy-date #{__('Date of your bankruptcy assignment')}
+
+          span#dobHint.cra-form-message #{__('For Example:')} 21 05 2006
+
+          if errors
+            ul
+              each err in errors
+                - if (err.param.startsWith("dob"))
+                  li
+                    +validationMessage(err.msg, err.param)
+
+          div.w-1-4.d--ib
+            label(for='dobDay') #{__('Day')}
+            input#dobDay(name='dobDay', type='text', value=dobDay, autocomplete='bday-day', aria-describedby=(errors && errors.dobDay ? 'dobDay-error' : false))
+
+          div.w-1-4.d--ib
+            label(for='dobMonth') #{__('Month')}
+            input#dobMonth(name='dobMonth', type='text', value=dobMonth, autocomplete='bday-month', aria-describedby=(errors && errors.dobMonth ? 'dobMonth-error' : false))
+
+          div.w-1-2.d--ib
+            label(for='dobYear') #{__('Year')}
+            input#dobYear(name='dobYear', type='text', value=dobYear, autocomplete='bday-year', aria-describedby=(errors && errors.dobYear ? 'dobYear-error' : false))
+
+      div(class={['has-error']: errors && errors.trusteeLastName && errors.trusteeFirstName})
```

Comment:
I think take the classname away. So leave this as just a div with no `has-error` class logic.